camunda-formio-tasklist-vue/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "camunda-formio-tasklist-vue",
-  "version": "1.0.15",
+  "version": "1.0.16",
```

Comment:
![image](https://user-images.githubusercontent.com/70306694/147644956-b90c865a-77fa-40f0-82f9-3785852177a0.png)

You can add your name here as a contributor

Reply:
added name